### DISCUSSION TITLE
Scratch Addons “getting started” tutorial video

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#6746] [Ideas]
For some users, Scratch Addons can be slightly unintuitive. Depending on how well you find your way around on your own, it can take a while to set up Scratch Addons the way you like it. Perhaps if someone could quote some feedback messages it'd help put that fact into perspective.

It might be nice if, on the welcome page, there was a video (probably uploaded to YouTube and embedded on the page) that explained how to navigate and use Scratch Addons:

1. Show how to get to the settings page from the extension's popup window, briefly explaining the popup as well.
2. Explain what addons are and how to enable, disable, and search for/find them.
3. Demonstrate an addon.
4. Show how to configure addons.
5. Go over how to quickly toggle and configure addons from the extension's popup window.

I know we have a welcome page with basic quickstart instructions, but video guidance works better for some people and the instructions that are currently written on the welcome page are very brief and don't seem to mention the amazing search feature.

It would be nice if the guide was still somehow available in text form as well, just in case, IDK, YouTube is blocked on someone's device or something.

@Samq64 would probably make for a pretty good video creator, just make sure to somewhat follow the steps above to cover everything users need to know.

**Comment by Samq64** [comment]
I might get around to it eventually but my setup isn't ideal and i don't really feel like it. 

I also haven't started the changelog video yet.

**Comment by WorldLanguages** [comment]
@DNin01 There's quite a few related issues to this. I remember you once suggested an overlay the first time you visit the settings page that would be quite similar to this, but without the video component.

**Reply by DNin01** [reply]
That was in #6099, right?

**Reply by WorldLanguages** [reply]
Looks like it.

**Comment by the-trumpeter** [comment]
If we do that then can we make 'How to make a basic addon walkthrough' videos as well?

**Reply by WorldLanguages** [reply]
I'm not sure if we can fit all implicit knowledge needed into a single video.

**Reply by the-trumpeter** [reply]
true
im already screaming at javascript

**Reply by the-trumpeter** [reply]
maybe just a walkthrough of making a button on the menu bar or something
